### DISCUSSION TITLE
Code completion engine has stopped working

### DISCUSSION THREAD

**Discussion by DamoM73** [pybricks/support#1432] [Q&A]
I am a secondary school teacher.

In my class I have some students who have the **Code completion engine has stopped working** error. Since some of these students also have dyslexia, the code completion feature is an import assistive technology.

This is occurring on both mac and Windows laptops. Students are mostly using Chrome, but the problem persists on Edge, Safari and Opera.

I am looking for some trouble shoot suggestions.

If I could also have the url for the Code completion engine, so I can ensure that the school firewal is not blocking it.

**Comment by dlech** [comment]
Which version of Pybricks are you using?

Can you share the logs from the Chrome developer tools? Press <kbd>F12</kbd> to open the developer tools. Check the "Console" and "Network" tabs.

**Comment by afarago** [comment]
I had this problem as well with the kids in the past (beta IDE, installed as a app, windows 11, 2024-Jan latest version that time).

We have put the laptop to sleep, and continued work a few days later when this error came.
I closed and reopened the IDE and it fixed the error,
